src/reference-manual/statements.qmd

```diff
@@ -365,42 +366,70 @@ To access accumulated log density up to the current execution point,
 the function `target()` may be used.
 
 
-## Sampling statements {#sampling-statements.section}
+## Distribution statements {#sampling-statements.section}
 
-Stan supports writing probability statements also in sampling
-notation, such as
+Stan supports writing probability statements also using distribution
+statements, for example
 
 ```stan
 y ~ normal(mu, sigma);
+mu ~ normal(0, 10);
+sigma ~ normal(0, 1);
 ```
+The symbol $\sim$ is called tilde. Due to historical reasons, the distribution statements used to be called "sampling statements" in Stan, but that term is not recommended anymore as it is less accurate description.
 
-The name "sampling statement" is meant to be suggestive, not
-interpreted literally.  Conceptually, the variable `y`, which may
-be an unknown parameter or known, modeled data, is being declared
-to have the distribution indicated by the right-hand side of the
-sampling statement. The above statement can read as "y is distributed
-as normal distribution with parameters mu and sigma.". The symbol `~`
-is called tilde.
+In general, we can read $\sim$ as "is distributed as," and overall this notation is used as a shorthand for defining distributions as
+$$
```

Comment:
I missed that this referred to the previous Stan example, so I think it'd help to add that here.